numpy/core/include/numpy/npy_math.h

```diff
@@ -145,6 +145,24 @@ NPY_INLINE static float __npy_nzerof(void)
 #define NPY_COEFF_Q4_LOGf 1.546476374983906719538e-01f                          
 #define NPY_COEFF_Q5_LOGf 5.875095403124574342950e-03f 
 
+/*
+ * Integer functions.
+ */
+npy_uint npy_gcdu(npy_uint a, npy_uint b);
+npy_uint npy_lcmu(npy_uint a, npy_uint b);
+npy_ulong npy_gcdul(npy_ulong a, npy_ulong b);
+npy_ulong npy_lcmul(npy_ulong a, npy_ulong b);
+npy_ulonglong npy_gcdull(npy_ulonglong a, npy_ulonglong b);
+npy_ulonglong npy_lcmull(npy_ulonglong a, npy_ulonglong b);
+
+npy_int npy_gcd(npy_int a, npy_int b);
+npy_int npy_lcm(npy_int a, npy_int b);
+npy_long npy_gcdl(npy_long a, npy_long b);
+npy_long npy_lcml(npy_long a, npy_long b);
+npy_longlong npy_gcdll(npy_longlong a, npy_longlong b);
+npy_longlong npy_lcmll(npy_longlong a, npy_longlong b);
```

Comment:
needs `NPY_INPLACE` like all the other functions in this file.

Reply:
Thanks!